site/gatsby-site/src/pages/incidents/new.js

```diff
@@ -85,6 +96,20 @@ function NewIncidentPage() {
 
       await insertIncident({ variables: { incident: newIncident } });
 
+      // Set the user as the last modifier
+      newIncident.modifiedBy =
+        user && user.customData.first_name && user.customData.last_name ? user.id : '';
```

Comment:
Should we make it a requirement to have first and last names set to be able to edit stuff instead of doing this? As it is, we'll lose track of who made the change.

Reply:
This condition is to know if the user is real, not an anonymous user from Realm. Because the `user` variable is always defined on our app.
The first and last names are not needed at this time, we only need the user Id

Reply:
Fixed